src/config.c

```diff
@@ -3439,6 +3439,55 @@ void addModuleNumericConfig(sds name, sds alias, int flags, void *privdata, long
     }
 }
 
+int moduleServerConfigSet(client *c, sds name, sds value) {
+    standardConfig *config = lookupConfig(name);
+
+    if (!config) return 0;
+
+    if (config->flags & MODULE_CONFIG) return 0;
```

Comment:
> The Server prefix indicates that only core config parameters can be accessed. This was done for the reason we already have module configs API. This may be subject to discussion so we can change it.

is this sentence about this line?
i think it's wrong.
one module should be able to query configs of another module, there's no reason to block it.